src/libraries/System.Diagnostics.DiagnosticSource/tests/MetricsTests.cs

```diff
@@ -894,6 +894,28 @@ public void MultipleListenersTest()
             }).Dispose();
         }
 
+        [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
+        public void NullMeasurementEventCallbackTest()
+        {
+             RemoteExecutor.Invoke(() => {
+                Meter meter = new Meter("NullMeasurementEventCallbackTest");
+
+                Counter<int> counter = meter.CreateCounter<int>("Counter");
+
+                MeterListener listener = new MeterListener();
+
+                listener.InstrumentPublished = (theInstrument, theListener) => theListener.EnableMeasurementEvents(theInstrument, theInstrument);
+
+                listener.SetMeasurementEventCallback<int>(null);
```

Comment:
>listener.SetMeasurementEventCallback<int>(null); [](http://example.com/codeflow?start=16&length=48)

Could you modify the test a little bit by adding non-null callback and validate this callback is called, then set the callback o null and then validate the original callback not called after that? Thanks for your help.

Reply:
I change it